lesson07/assignment/vue-router-01/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+<main class="main">
+  <div v-for="feature in features" class="jumbotron jumbotron-fluid" v-bind:key="feature.id">
```

Comment:
The "key" attribute should be unique for each element looped through. Since feature items don't have a unique identifier, you can use the index of the loop, by changing the "for" syntax, and then referencing the index value in the key. 
```
<div v-for="(feature, index) in features" class="jumbotron jumbotron-fluid" v-bind:key="index">
```

Reply:
Pushed the changes ,please check